fc-java-core/src/main/java/com/aliyun/fc/runtime/FunctionParam.java

```diff
@@ -0,0 +1,28 @@
+package com.aliyun.fc.runtime;
+
+/**
+ * This provides the information related to the invoking function
+ */
+public interface FunctionParam {
+
+    /**
+     * Gets the name of the function being executed.
+     */
+    public String getFunctionName();
+
+    /**
+     * Gets the fully qualified name of the handler being executed.
+     */
+    public String getFunctionHandler();
+
+    /**
+     * Gets the execution time limit of the function
+     */
+    public int getExecTimeLimitInMillis();
+
+    /**
+     * Gets the maximum memory size configured for the function
+     */
+    public int getMemoryLimitInMB();
+
```

Comment:
Remove blank line

Reply:
Actually, that seems to be the convention for all interfaces. I checked the Lambda's interface.